dev/io.openliberty.org.hibernate.validator.cdi.7.0/bnd.overrides

```diff
@@ -0,0 +1,20 @@
+-include= ~../cnf/resources/bnd/rejar.props
+
+bVersion=1.0
+Bundle-SymbolicName: io.openliberty.org.hibernate.validator.cdi.7.0
+
+# Something that wildfly/thorntail apparently uses but blows up on Liberty
+Fragment-Host:
+
+Import-Package: \
+  org.apache.el,\
+  *
+
+Export-Package: \
+  org.hibernate.validator.cdi.*;version="7.0.0.Alpha2"
+
+Include-Resource: \
+  @${repo;org.hibernate.validator:hibernate-validator-cdi;7.0.0.Alpha2}!/META-INF/services/*,\
+  META-INF=resources/META-INF
+
+-cdiannotations:
```

Comment:
Is this clause intentionally left blank? If so, it can be removed

Reply:
It looks to be needed. I took it out and the Manifest changed.  Here is some of the background info I was given:
`When we updated to Bnd 4.3.1 the default behavior changed. Bnd is finding and processing CDI beans in  bundles. The default for Bnd is -cdiannotations: * which means process all classes for CDI beans. You can disable that if you don't use OSGi's CDI Integration spec by using -cdiannotations:`